### PR TITLE
0.4.1 docs update

### PR DESCRIPTION
**Pull request by finglis**

### CODE REVIEW

File: docs/intro/formats.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Both libraries have their own distinct advantages:
 - OpenSlide is typically fast, lightweight and programmer-friendly (especially through Python), but is restricted to *only* 2D, 8-bit, RGB images.
 - Bio-Formats can support a much wider range of image types, including multidimensional data (e.g. z-stacks, time series and multiplexed images).
 
-Bio-Formats supports most images that can be read by OpenSlide, but not all - and therefore QuPath continues to include both libraries.
+Bio-Formats supports most images that can be read by OpenSlide, but not all - and therefore QuPath continues to include both libraries. However, some formats that are supported by Bio-formats or OpenSlide are unfortunately not usable in QuPath due to various caveats.
```
Comment (petebankhead) [inline docs/intro/formats.md:26]: Suggest rewording:

> Bio-Formats supports most images that can be read by OpenSlide, but not all - and therefore QuPath continues to include both libraries.
> 
> However, file formats are tricky and tend to have lots of variants. Just because a format is listed as being supported by Bio-Formats or OpenSlide doesn't mean files in that format will always open (properly) in QuPath. The following sections contain some extra details and caveats, partly based on user feedback about what does and doesn't work.

Reason: the problems generally aren't things that can be resolved on the QuPath side, but because rather there are variations among the formats themselves that don't always work with Bio-Formats/OpenSlide.

File: docs/starting/first_steps.md [hunk 2]
```diff
@@ -34,6 +52,27 @@ This works for most other file types that QuPath can handle - not only images.
...
+:::{figure} images/steps_image_type.jpg
+:align: center
+:class: shadow-image
+:width: 75%
+
+Option window for setting the image type in QuPath.
+:::
+
+:::{important}
+QuPath can automatically estimate the image type for you, if you choose this option for *Set image type* in under {menuselection}`Edit --> Preferences...` {{ icon_cog }}.
```
Comment (petebankhead) [inline docs/starting/first_steps.md:68]: I think we should highlight first that the user should press 'Show details'. I think you can use
```
{guilabel}`Show details`
```
to highlight the fact it's a UI button.

You could note that option can always be changed in the preferences, but I think flagging that 'Show details' buttons really are useful could help more.

File: docs/starting/first_steps.md [hunk 3]
```diff
@@ -34,6 +52,27 @@ This works for most other file types that QuPath can handle - not only images.
...
+However, in case the estimate is wrong you should be aware the option exists - and should always be checked.
+:::
+
+:::{tip}
+Should you need to change the image type retrospectively, it's possible to do so in the **Image** tab within the **Analysis panel** mentioned below.
```
Comment (petebankhead) [inline docs/starting/first_steps.md:73]: Mention you should double-click on the type

File: docs/tutorials/density_maps.md [hunk 4]
```diff
@@ -1,8 +1,6 @@
 # Density maps
 
-**Density maps** are a new concept introduced in QuPath v0.3.0.
-
-They were originally created to help find 'hotspots' for specific applications in pathology (e.g. tumor budding, Ki67 scoring).
+Density maps were originally created to help find 'hotspots' for specific applications in pathology (e.g. tumor budding, Ki67 scoring).
```
Comment (petebankhead) [inline docs/tutorials/density_maps.md:3]: Suggest

> Density maps were introduced in QuPath v0.3.0. They were originally...

Some people reading the docs may be reluctant upgraders, so I think it helps to retain references to when something was added (but ideally with better wording than mine, since mine doesn't date well :)

File: docs/tutorials/exporting_measurements.md [hunk 5]
```diff
@@ -45,14 +45,14 @@ If your analysis involves combining measurements from different images, it is re
 
 The cleanest way to export different types of measurements in QuPath across multiple images is with the **Measurement Exporter**.
 
-Provided that your images are stored in a {doc}`project <../tutorials/projects>`, you can access it through {menuselection}`Measure --> Export measurements` (as well as {{ icon_table }} --> Export measurements).
+Provided that your images are stored in a {doc}`project <../tutorials/projects>`, you can access it through {menuselection}`Measure --> Export measurements`.
 
 :::{figure} images/measurement_exporter.png
```
Comment (petebankhead) [inline docs/tutorials/exporting_measurements.md:50]: This doesn't display any more because the updated dialog is a .jpg rather than .png.

Since there are so many images, I try to use

* PNG for dialogs and graphics with lots of flat colors (since they compress well, and look sharper)
* JPEG for screenshots that show an image with more variation (so a PNG would be much larger)

So I'd change the screenshot to a PNG

File: docs/tutorials/thresholding.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ A *thresholder* in QuPath is something that can apply a threshold to an image wh
 
 ### Thresholds in action
 
-You can create a thresholder using the *Create thresholder* command.
+You can create a thresholder using the *Create thresholder* command within {menuselection}`Classify --> Pixel classification --> Create thresholder.`
```
Comment (petebankhead) [inline docs/tutorials/thresholding.md:37]: Suggest:
> You can create a thresholder using the {menuselection}`Classify --> Pixel classification --> Create thresholder`>  command.

Reason: we should probably use 'menuselection' consistently, since we can protect text in there from automatic translaters (with [a bit of extra effort](https://twitter.com/petebankhead/status/1614629700604674049?s=20&t=Xk9rzyctAigy2FdRWozWdQ)).

File: docs/reference/shortcuts.md [hunk 7]
```diff
@@ -34,7 +34,7 @@ These shortcuts only work whenever the viewer is 'in focus', i.e. it is the last
 ```
 
 :::{sidebar} Note for Linux users
-Replace {kbd}`Alt` with {kbd}`Alt + Super` (thanks to [Philipp Tschandl for this](https://groups.google.com/d/msg/qupath-users/QB03tHjRrZs/od-nL_9SBQAJ)).
+Replace {kbd}`Alt` with {kbd}`Alt + Super`.
```
Comment (petebankhead) [inline docs/reference/shortcuts.md:37]: Suggest:

> You may need to replace {kbd}`Alt` with {kbd}`Alt + Super`

(Since I don't know if you always do...)

**Review by petebankhead — CHANGES_REQUESTED** [review]
Thanks! Looks good, I've added some suggested changes before merging (happy to discuss if any are controversial, but I think they're all small)

File: docs/tutorials/exporting_measurements.md (hunk 5 above)
Comment (finglis) [reply docs/tutorials/exporting_measurements.md:50]: Ah so that's the reason there was a mix! Thanks

File: docs/tutorials/thresholding.md [hunk 8]
```diff
@@ -34,7 +34,7 @@ A *thresholder* in QuPath is something that can apply a threshold to an image wh
 
 ### Thresholds in action
 
-You can create a thresholder using the *Create thresholder* command.
+You can create a thresholder using the {menuselection}Classify --> Pixel classification --> Create thresholder> command.
```
Comment (petebankhead) [inline docs/tutorials/thresholding.md:37]: This doesn't display properly, it needs to be
```
{menuselection}`Classify --> Pixel classification --> Create thresholder`
```

File: .vscode/settings.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.ignoreWords": ["menuselection"]
+}
```
Comment (petebankhead) [inline .vscode/settings.json:3]: Please add a line with 
```
.vscode
```
to the `.gitignore` file (or possibly `.vscode/` - whichever one excludes the directory, to try to avoid further IDE-specific files being added later)

**Review by petebankhead — CHANGES_REQUESTED** [review]

File: docs/tutorials/thresholding.md (hunk 8 above)
Comment (finglis) [reply docs/tutorials/thresholding.md]: section amended to 
`{menuselection}`Classify --> Pixel classification --> Create thresholder``
